### ISSUE TITLE
Settings text white

### ISSUE THREAD

**Issue by alexismdr** [Xinerki/kgv-phone#10]
When I go to the parameters, the text of the two unselected parameters is white and therefore invisible because white on white.
![Image1](https://user-images.githubusercontent.com/32903418/115939028-fa87f680-a49c-11eb-84a7-5f832cbe2e78.png)
But when I select a parameter and that I deselect it, this one goes back to black.
![Image2](https://user-images.githubusercontent.com/32903418/115939041-083d7c00-a49d-11eb-8461-bb9a7fd0f284.png)
When I go back to the top of the parameters by going all the way down, the parameters become white on white again.
![Image1](https://user-images.githubusercontent.com/32903418/115939044-0a9fd600-a49d-11eb-9a7e-be3fcea491e0.png)

I don't know if the problem is with me or with the kgv-phone...
I would be very grateful for any help or advice!

**Comment by Xinerki** [comment]
this is a known issue that wasn't always a thing, it's hard to say what causes it, probably gonna need tons of trial and error

**Comment by alexismdr** [comment]
I am actually doing a lot of research on this scaleform in order to integrate it into my server. I found that this problem can be solved by putting SetColourOfNextTextComponent(2) before BeginTextComponent("STRING") when using SET_DATA_SLOT. Basically this forces the next string to be hudcolour 2 (HUD_COLOUR_BLACK). But this is clearly not the right solution because it does not work on app 6 (messageList) and the first item in the list in the app is therefore also in black which is a problem because it should be white : we must then solve the problem by simulating an input down and up of the user by using SetInputEvent(scaleform, 3) SetInputEvent(scaleform, 1). 
I've explored the decompiled GTA scripts a lot and still haven't found anything, I'm starting to get a little discouraged.
